bridges/DealabsBridge.php

```diff
@@ -458,19 +551,189 @@ private function relativeDateToTimestamp($str) {
 		return $date->getTimestamp();
 	}
 
+	/**
+	 * Returns the RSS Feed title according to the parameters
+	 * @return string the RSS feed Tiyle
+	 */
 	public function getName(){
 		switch($this->queriedContext) {
-			case 'Recherche par Mot(s) clé(s)':
-				return self::NAME . ' - Recherche : '. $this->getInput('q');
-				break;
-			case 'Deals par groupe':
-				$values = self::PARAMETERS['Deals par groupe']['groupe']['values'];
-				$groupe = array_search($this->getInput('groupe'), $values);
-				return self::NAME . ' - Groupe : '. $groupe;
-				break;
-			default: // Return default value
-				return self::NAME;
+		case $this->i8n('context-keyword'):
+			return $this->i8n('bridge-name') . ' - '. $this->i8n('title-keyword') .' : '. $this->getInput('q');
+			break;
+		case $this->i8n('context-group'):
+			$values = static::PARAMETERS[$this->i8n('context-group')]['group']['values'];
+			$group = array_search($this->getInput('group'), $values);
+			return $this->i8n('bridge-name') . ' - '. $this->i8n('title-group'). ' : '. $group;
+			break;
+		default: // Return default value
+			return static::NAME;
+		}
+	}
+
+
+
+	/**
+	 * This is some "localisation" function that returns the needed content using
+	 * the "$lang" class variable in the local class
+	 * @return various the local content needed
+	 */
+	public function i8n($key)
+	{
+		if (array_key_exists($key, $this->lang)) {
+			return $this->lang[$key];
+		} else {
+			return null;
 		}
 	}
 
 }
+
+/*
+
+class ExampleDealsBridge extends PepperBridgeAbstract {
```

Comment:
The example should probably not be in the file. You should create a wiki page for this.